tools/cloud-build/daily-tests/integration-group-3.yaml

```diff
@@ -123,7 +123,11 @@ steps:
     set -x -e
     BUILD_ID_FULL=$BUILD_ID
     BUILD_ID_SHORT=$${BUILD_ID_FULL:0:6}
+    OMNIA_EXAMPLE=community/examples/omnia-cluster.yaml
 
-    ansible-playbook tools/cloud-build/daily-tests/ansible_playbooks/packer-integration-test.yml \
+    # Inject the build ID into the name prefix of the vm-instance modules to avoid naming collisions
+    sed -i "s/name_prefix: \(.*\)/name_prefix: \1-$${BUILD_ID_SHORT}/" $${OMNIA_EXAMPLE}
```

Comment:
nit for future: This would have been easier to review if the rearrange had been done in a separate commit.

Reply:
I was hoping the git diff would be cleaner, but it didn't turn out that way. I'll keep that in mind in the future.